File: p4/4.2.py
from funk import *
max = 5

def sprczyw(n, tab, size):
    for i in range(size):
        if tab[i] == n:
            return "tak"
    return "nie"

def main():
    re = True
    while re == True:
        size = inputh("Jaki ma być rozmiar tablicy:")
        tab = [0,]*size
        print("podajemy ? (T/N)")
        ans = getin()
        if ans == b't':
            czytajt(tab,size)
        else:
            lost(tab,size)
        piszt(tab,size)
        szuk = inputh("jako liczbe chcesz sprawdzic?:")
        print(f"czy jest:{sprczyw(szuk,tab,size)}")
        re = jeszcze()

if __name__ == "__main__":
    main()

File: s4/4.2.cpp
#include "funk.h"
#define max 100
using namespace std;

string sprczyw(int n,int tab[max] ,int size){
    for(int i=0;i<size;i++){
        if( tab[i] == n){return "tak";}
    }
    return "nie";

}
int main(){
    srand(time(NULL));
    bool re = true;
    do{
        int size = input("Jaki ma byc rozmiar tablicy: ");        
        int tab[size];
        cout<<"podajemy ? t or n\n";
        char ans = getin();
        if(ans == 't'){
            czytajt(tab, size);
        }
        else{
            lost(tab, size);
        }
        piszt(tab, size);
        int szuk = input("jako liczbe chcesz sprawdzic?: ");
        cout<<"czy jest:"<<sprczyw(szuk,tab,size);
        jeszcze(re);
    }while(re==true);
    return 0;
}
